.github/workflows/lint.yaml

```diff
@@ -34,12 +37,12 @@ jobs:
         id: list-changed
         run: |
           ## If executed with debug this won't work anymore.
-          changed=$(ct --target-branch ${{ github.event.pull_request.head.ref }} --config ./.github/configs/ct-lint.yaml list-changed)
+          changed=$(ct --target-branch develop --config ./.github/configs/ct-lint.yaml list-changed)
```

Comment:
why did we change it to develop only?

Reply:
it can be run against one branch only. we need to add another flow for `main` to run when a PR that targets `main` is open 
previous set-up didn't work correctly.